packages/database/.babelrc.js

```diff
@@ -0,0 +1,38 @@
+const env = {
+  test: {
+    plugins: ['istanbul'],
+  },
+};
+
+const includedDateOffset = 90 * 24 * 60 * 60 * 1000; // include migrations up to 90 days old
```

Comment:
Because meditrak-server needs to run the babelified migrations, we needed to take them off the ignore list. However, there are so many that it makes the compilation takes 3x the time. This js based babel config has a custom ignore function, that ignores any migrations which are older than 90 days.

Reply:
Clever!!